server/tests/metadata.rs

```diff
@@ -73,7 +101,22 @@ mod entities_metadata_query {
         )
         .execute()
         .await
-        .should_be_ok_with_body(json!({
+        .should_be_ok_with(|actual_response| {
+        let actual_date_created = DateTime::parse_from_rfc3339(
+          &actual_response["entities"][0]["mainEntityOfPage"]["dateCreated"]
+            .as_str()
+            .unwrap()
+        ).unwrap();
+        assert!(Utc::now() < actual_date_created + Duration::seconds(1));
+
+        let actual_date_modified = DateTime::parse_from_rfc3339(
+          &actual_response["entities"][0]["mainEntityOfPage"]["dateCreated"]
+            .as_str()
+            .unwrap()
+        ).unwrap();
+        assert!(Utc::now() < actual_date_modified + Duration::seconds(1));
+
+        let expected_response = json!({
```

Comment:
Seems like something we can put in another function (I see duplicate code here) -> Can be a follow up issue though